_posts/2021-02-22-tiaas-Mali-Salmon.md

```diff
@@ -0,0 +1,15 @@
+---
+site: [pasteur, freiburg, erasmusmc, elixir-it, belgium, genouest]
+title: 'Training Infrastructure Feedback from Mali Salmon-Divon'
+tags: [training, TIaaS]
+supporters:
+- galaxy-europe
+author: Mali Salmon-Divon
```

Comment:
Will this work? I think this only works for registered users, no?

Reply:
We have done it for other TIaaS feedbacks as well, so I guess it won't be like [this](https://galaxyproject.eu/posts/2021/01/30/cnv-and-vc-updates/) but like [this](https://galaxyproject.eu/posts/2021/01/28/tiaas_Lachlan/) that has the author too. In short, not displayed on the website but it's in the md.